minimisers/DREAM/functions/GenParSet.m

```diff
@@ -1,4 +1,4 @@
-function ParSet = GenParSet(chain);
+function ParSet = GenParSet(chain)
```

Comment:
There is an identical GenParSet in postprocessing folder. I think we can delete one